ios/instruments/screenshot.go

```diff
@@ -62,12 +95,12 @@ func startConversionQueue() {
 	}
 }
 
-func startScreenshotting(conn *Connection) {
+func startScreenshotting(conn *ScreenshotService) {
 	for {
 		start := time.Now()
 		pngBytes, err := conn.TakeScreenshot()
 		if err != nil {
-			log.Fatal("screenshotr failed", err)
+			log.Fatal("Screenshot failed", err)
```

Comment:
can you find a way to remove logFatal? Don't know why I added this, but the library should not log.Fatal